model/artifact/artifact_file.go

```diff
@@ -132,7 +132,7 @@ func presignFile(ctx context.Context, file File) (string, error) {
 			grip.Debug(message.Fields{
 				"message":    "presigning with IRSA failed",
 				"ticket":     "DEVPROD-13970",
-				"error":      err,
+				"error":      err.Error(),
```

Comment:
nit: usually grip logs are written as `grip.Debug(message.WrapError(err, message.Fields{ /* other fields... */ ))` to 1. only log when there's an error and 2. avoid JSON marshalling problems since error types are typically don't have logic/JSON tags to marshal to JSON.

Reply:
I moved it to use it. I checked the codebase but it looks like a mix of checking the error anyways and then logging it and just logging it and letting grip not log it from [this helper func](https://github.com/mongodb/grip/blob/main/message/error_message.go#L85). Do you know which one is more common? This looked nicer so I picked it but not sure if it's our standard